### ISSUE TITLE
torch.linalg was deprecated 

### ISSUE THREAD

**Issue by kamo-naoyuki** [fgnt/ci_sdr#4]
Could you update to use `torch.linalg.sovle` for pytorch>=1.9?

```
lib/python3.8/site-packages/ci_sdr/pt/wiener_filter.py:371: in wiener_filter_predict_single_input
    w, _ = torch.solve(p[..., None], R)
```

```
    def solve(input: Tensor, A: Tensor, *, out=None) -> Tuple[Tensor, Tensor]:
>       raise RuntimeError(
            "This function was deprecated since version 1.9 and is now removed. Please use the `torch.linalg.solve` function instead.",
        )
E       RuntimeError: This function was deprecated since version 1.9 and is now removed. Please use the `torch.linalg.solve` function instead.
```

**Comment by boeddeker** [comment]
Thank you for reporting. I will fix it this week.

**Comment by boeddeker** [comment]
Sorry for the delay, it took more time than expected (Python3.7 and recent Torch yield a Segmentation Fault).
The fix is in #5, and I uploaded it to PyPI as version 0.0.2.

**Comment by kamo-naoyuki** [comment]
Thank you!
